packages/strapi-plugin-graphql/README.md

```diff
@@ -1 +1,6 @@
-# Strapi plugin
+# Strapi GraphQL
+
+This plugin will add GraphQL functionality to your app.
+By default it will provide you with all the default CRUD methods.
```

Comment:
Not entirely true -yet- as there are still some REST CRUD methods not implemented in GraphQL yet.

I would say `most` instead of `all` here.

Reply:
@derrickmehaffy ohh cool, any specific or common ones that are not yet implemented to point out? 
386984f08

Reply:
File updates for the upload plugin (changing metadata) is one I just recently have seen an issue for.

Reply:
> File updates for the upload plugin (changing metadata) is one I just recently have seen an issue for.

Ahh, makes sense, ok I don't think I'll be able to provide more details then, so leaving my edits at this then 👍